redbeams/src/main/java/com/sequenceiq/redbeams/controller/v4/databaseserver/DatabaseServerV4Controller.java

```diff
@@ -272,4 +279,10 @@ public UpgradeDatabaseServerV4Response validateUpgrade(@TenantAwareParam @NotEmp
         UpgradeDatabaseRequest upgradeDatabaseRequest = upgradeDatabaseServerV4RequestConverter.convert(request);
         return upgradeDatabaseServerV4ResponseConverter.convert(redbeamsUpgradeService.validateUpgradeDatabaseServer(crn, upgradeDatabaseRequest));
     }
+
+    @Override
+    @InternalOnly
+    public FlowIdentifier rotateSecret(RotateDatabaseServerSecretV4Request request, String initiatorUserCrn) {
```

Comment:
`@InitiatorUserCrn` annotation is missing

Reply:
Thanks